indexing-service/src/test/java/org/apache/druid/indexing/common/task/IngestionTestBase.java

```diff
@@ -114,6 +119,7 @@ public abstract class IngestionTestBase extends InitializedNullHandlingTest
   private SegmentsMetadataManager segmentsMetadataManager;
   private TaskLockbox lockbox;
   private File baseDir;
+  protected File reportsFile;
```

Comment:
We don't seem to be writing to this file anywhere.

Reply:
It's written in `CompactionTask` class.

Reply:
If it's written only in `CompactionTask` class, why does it need to be a part of the base class?

Reply:
Ah, never mind, just saw that it is being used in `AbstractParallelTaskTestAsWell`.

However, could you make this field private and expose a `protected getReportsFile()` method? We don't want any test to reassign this variable (unless the test needs to write out multiple different reports, but that doesn't seem to be the case right now).